draft-ietf-v6ops-464xlaton.xml

```diff
@@ -280,38 +279,98 @@ CLAT IPv6 Addresses
-This document updates <xref target="RFC6877"/> recommendations.
-More specifically, obtaining a /64 just for translating is NOT RECOMMENDED even in a dedicated prefix model.
-However, an instance MAY obtain a dedicated IPv6 address for each CLAT IPv4 address.
+The node SHOULD treat its CLAT IPv6 addresses as any other IPv6 address and comply with <xref target="RFC4861"/> and <xref target="RFC4862"/>. In particular:
 </t>
+<ul>
+<li>
 <t>
-The node MUST treat the CLAT addresses as any other address and comply with <xref target="RFC4861"/> and <xref target="RFC4862"/>. In particular:
+Performing Duplicate Address Detection for each dedicated CLAT address (Section 5.4 of <xref target="RFC4862"/>);
 </t>
 <ul>
 <li>
 <t>
-Performing Duplicate Address Detection for each dedicated CLAT address (Section 5.4 of <xref target="RFC4862"/>);
+Justification: performing DAD minimizes in the unlikely event of address collision.
+Additionaly, network infrastructure devices mandate a DAD packet from the client before enabling network access.
 </t>
 </li>
+</ul>
+</li>
+<li>
+<t> Processing received unicast Neighbor Solicitations (NSes) as well as multicast ones sent to the solicited-node multicast address (<xref target="RFC4861"/>) for the node CLAT addresses.
+</t>
+<ul>
 <li>
-<t> Processing received Neighbor Solicitations sent to the solicited-node multicast address (<xref target="RFC4861"/>) for the node CLAT addresses.</t>
+<t>
+Justification: If a node doesn't respond to unicast NSes, anytime the first-hop router gets a packet for the CLAT address and its Neighbor Cache entry is 'STALE' (Section 7.3.2 of <xref target="RFC4861"/>), the Neighbor Unreachability Detection process (Section 7.3.3 of <xref target="RFC4861"/>) will delete that CLAT address's cache entry. This forces the address resolution process to restart from scratch. Until resolution finishes, traffic for the CLAT address might drop, leading to a degraded user experience, especially for applications sensitive to jitter and packet loss.
+</t>
 </li>
+</ul>
+</li> 
 <li>
 <t>Sending Gratuitous Neighbor Advertisements (<xref target="RFC9131"/>) for the CLAT addresses.</t>
+<ul>
+<li>
+<t> Jutification: not following this recommendation leads to user-visible packet loss when the CLAT instance starts receiving traffic after period of inactivity, or when connected to the network for the first time. The problem is discussed in Section 2 of <xref  target="RFC9131"/>.</t>
+</li>
+</ul>
 </li>
 <li>
-Registering the CLAT addresses using DHCPv6 (<xref target="RFC9686"/>), if the network supports the registration.
+<t>
+Registering the CLAT addresses assigned via SLAAC or statically using DHCPv6 (<xref target="RFC9686"/>), if the network supports the registration.
+</t>
+<ul> 
+<li> 
+<t>
+Justification: not registering CLAT addresses creates a significant blid spot for network operators.
+This lack of visibility complicates troubleshooting and forensics, as discussed in <xref target="RFC9686"/>.
```

Comment:
```suggestion
Justification: not registering CLAT addresses reduces traffic visibility for network operators, which complicates troubleshooting and forensics, as discussed in <xref target="RFC9686"/>.
```

Reply:
If you do not like this re-write, please note that there's a typo in the original to be fixed:

s/blid/blind/